fix: #375 fixes The Footer Section and give  a good look

officeneerajsaini opened the pull request:
The footer section requires some design enhancements, including alignment issues and excess space below the footer.

**GitHub Issue:** #375 

**Summary**:

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)** : No


![Screenshot from 2024-02-27 12-27-47](https://github.com/json-schema-org/website/assets/118799941/1c278db5-990a-4dd3-8d27-3dabe5f9e1de)

![Screenshot from 2024-02-27 12-27-32](https://github.com/json-schema-org/website/assets/118799941/c81c765f-6934-4d9a-84b8-7e07ba0e972e)

![Screenshot from 2024-02-27 12-37-38](https://github.com/json-schema-org/website/assets/118799941/bf2e8325-f345-492d-80ce-86e5d49eb179)

Min2who commented:
Thanks for your PR @officeneerajsaini 

But the issue has already been resolved by #364

JeelRajodiya commented on pages/index.page.tsx:
These styles are changing the size of the slack and common room icons 

same as this PR -> https://github.com/json-schema-org/website/pull/372

![image](https://github.com/json-schema-org/website/assets/63534268/76bfc49b-0402-4e5b-9344-8479e511c5cb)

JeelRajodiya reviewed (commented):
Your code resolves the footer issue, 
but I think you have also included commits of your other PR ( [this PR](https://github.com/json-schema-org/website/pull/372) ) in your current PR. 

let's wait for the maintainer's comment

benjagm reviewed (changes requested):
Now I understand. This is a good enhancement. Thanks!

Maybe we need to make sure is centered horizontally. See image. 

<img width="990" alt="Screenshot 2024-02-28 at 10 59 54" src="https://github.com/json-schema-org/website/assets/40007659/6f025191-82f0-493d-8ec6-bbcbd2a3a587">

benjagm reviewed (changes requested):
Great job @officeneerajsaini 

I saw a comment by @JeelRajodiya regarding side effects in other areas . Can you please review that you are not adding or overwriting changes from others?

officeneerajsaini commented:
Yes, I removed some code and added new code as a replacement because the existing code was lengthy. I found a simpler way to address it, and I tested the changes before creating a pull request to ensure they functioned as well as, or even better than before. Is it considered improper to remove contributed code in this manner? Did I do something wrong?

officeneerajsaini commented:
![Screenshot from 2024-02-29 17-38-15](https://github.com/json-schema-org/website/assets/118799941/e2e39871-dfbe-4962-b2ec-87096629bf50)

![Screenshot from 2024-02-29 18-07-41](https://github.com/json-schema-org/website/assets/118799941/e5529d23-e53e-4f4e-ba77-592f188feb54)

benjagm reviewed (dismissed):
Great job! looks good!

officeneerajsaini commented:
@benjagm now all conflicts resolved

benjagm commented:
I am getting this error when checking again

![Screenshot 2024-03-03 at 16 47 23](https://github.com/json-schema-org/website/assets/40007659/5d724187-454e-416d-a9fa-6a264f42db68)

benjagm reviewed (changes requested):
Found this error after the last rebase:

https://github.com/json-schema-org/website/pull/376#issuecomment-1975202751

officeneerajsaini commented:
> Found this error after the last rebase:
> 
> [#376 (comment)](https://github.com/json-schema-org/website/pull/376#issuecomment-1975202751)
Okay @benjagm

benjagm commented:
@officeneerajsaini lets fix https://github.com/json-schema-org/website/pull/376#issuecomment-1975202751 to merge this asap.

officeneerajsaini commented:
Yes i need some time 😀✨
